code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -13,12 +13,27 @@
 	taste_description = "generic food"
 	taste_mult = 4
 	var/nutriment_factor = 1 * REAGENTS_METABOLISM
+	var/quality = 0	//affects mood, typically higher for mixed drinks with more complex recipes
 
 /datum/reagent/consumable/on_mob_life(mob/living/carbon/M)
 	current_cycle++
 	M.nutrition += nutriment_factor
 	holder.remove_reagent(src.id, metabolization_rate)
 
+/datum/reagent/consumable/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(method == INGEST)
+		if (quality && !M.has_trait(TRAIT_AGEUSIA))
+			switch(quality)
```

Comment:
because then this becomes more sensical if you have 4 defines

Reply:
makes sense, should I put them here on top of food_reagents or with the other food defines?

Reply:
You can put them on either depending on how localized you want it to be.